plume-models/src/safe_string.rs

```diff
@@ -25,17 +25,7 @@ lazy_static! {
             .add_tag_attributes(
                 "iframe",
                 [ "width", "height", "src", "frameborder" ].iter().map(|&v| v),
-            )
-            .add_tag_attributes(
-                "video",
-                [ "src", "title", "controls" ].iter(),
-            )
-            .add_tag_attributes(
-                "audio",
-                [ "src", "title", "controls" ].iter(),
```

Comment:
I think we should keep video and audio tags, So that people can insert media content in their posts